docker/Dockerfile

```diff
@@ -114,20 +114,103 @@ RUN mkdir -p /usr/src/tini && \
     make && \
     make install DESTDIR=/dist/tini && \
     mv /dist/tini/usr/local/bin/tini-static /dist/tini/usr/local/bin/tini
-
 #
 # Ubuntu 14.04 base
 #
-FROM ${MY_REGISTRY}ubuntu:trusty as ubuntu-14.04
+FROM ${MY_REGISTRY}ubuntu:trusty as ubuntu-14.04-base
 LABEL maintainer="Joshua Watt <Joshua.Watt@garmin.com>"
 
 # Install software required to add ppa's
-RUN apt-get -y update && apt-get -y install \
+RUN set -x && export DEBIAN_FRONTEND=noninteractive && apt-get -y update && apt-get -y install \
     python-software-properties \
-    software-properties-common
-
+    sudo \
+    locales \
+    python \
+    python3 \
+    software-properties-common && \
 # Add a non-ancient version of git
-RUN add-apt-repository -y ppa:git-core/ppa
+    add-apt-repository -y ppa:git-core/ppa && \
```

Comment:
Should this be in the base?

Reply:
yea. that should have already been added to the base image.

Reply:
oh, I see your point. I guess we can push that up to parent images.

Reply:
This is the "parent" (e.g. "-base") image? I was wondering if this should be pushed into the individual "-oe" images.

Reply:
I moved this up into the -oe images.